app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -186,6 +186,8 @@ describe('League Week Picks', () => {
     // Wait for the main content to be displayed
     await waitFor(() => {
       expect(screen.getByTestId('weekly-picks')).toBeInTheDocument();
+      expect(screen.getByTestId('week__week-number')).toHaveTextContent('Week 1');
+      expect(screen.getByTestId('week__entry-name')).toHaveTextContent('Entry 1');
```

Comment:
NIT: Is there a particular reason you named this `week__entry-name` instead of `week-number` or `week-entry-name`?

Reply:
@choir241 yes, Shashi requested this naming device